corehq/apps/cleanup/management/commands/couch_migrate.py

```diff
@@ -49,7 +49,7 @@ def handle(self, *args, **options):
 def _copy(config):
     # unfortunately the only couch view we have for this needs to go by domain
     # will be a bit slow
-    database = Domain.get_db()
+    database = get_db()
```

Comment:
Looks like this this management command is deprecated in favor of `doctypemigrations`'s `run_doctype_migration`.